templates/vector.toml.j2

```diff
@@ -16,12 +16,29 @@
 
 data_dir = "{{ vector_data_dir }}"
 
+{% for item in vector_parser_config %}
 # Input data. Change me to a valid input source.
-[sources.in]
-  type = "stdin"
+
+[sources.{{ application.name }}]
+  include = [ "{{ application.path_log }}" ]
+  type = "file"
+  start_at_beginning = true
+
+[transforms.{{ application.name }}_parser]
+  inputs = ["{{ application.name }}"]
+  type = "remap"
+  source = """{{ application.parse_vrl }}"""
 
 # Output data
-[sinks.out]
-  inputs   = ["in"]
-  type     = "console"
-  encoding = "text"
+[sinks.{{ application.name }}_output]
```

Comment:
I would prefer not to focusing configs in a role on such specific backend (since vector can store in multiple backends). Let's allow users taking their own decisions ;-)

Reply:
Edited back to the previous version.